templates/zerver/register.html

```diff
@@ -46,6 +46,39 @@
             {% endif %}
         </div>
     </div>
+
+  {% if creating_new_team %}
+        <div class="control-group">
+            <label for="id_team_name" class="control-label">{{ _('Team name') }}</label>
+            <div class="controls">
+                <input id="id_team_name" class="required" type="text"
+                    placeholder="E.g. Acme"
+                    name="realm_name" maxlength="100" />
+                    {% if form.realm_name.errors %}
+                        {% for error in form.realm_name.errors %}
+                        <div class="alert alert-error">{{ error }}</div>
+                        {% endfor %}
+                    {% endif %}
+                <br /><span class="small">(You can change this later on the admin page.)</span>
```

Comment:
We should tag this string for translations.  See http://zulip.readthedocs.io/en/latest/translating.html if you're not sure how to do that.